envs/ppc/esi-2024b-ppc.yml

```diff
@@ -0,0 +1,52 @@
+name: esi-2024b-ppc
+channels:
+  - https://ftp.osuosl.org/pub/open-ce/1.10.0
+  - https://opence.mit.edu
+  - https://ftp.osuosl.org/pub/open-ce/1.10.0/
```

Comment:
duplicate channel (both in concretized and regular yml)

Reply:
I think that from when I tried to recreate the kilosort environment. I followed all the steps, but because it was already added to the channels, it added it again. I can remove the additional one (I don't think it matters if it contains the / or not at the end right?)

Reply:
I just removed one of the duplications now.

Reply:
Great, thanks! And yes, on our cluster it shouldn't matter, since we set these channels globally anyway.